llvm/test/CodeGen/ARM/dagcombine-drop-flags-freeze.ll

```diff
@@ -0,0 +1,40 @@
+; NOTE: Assertions have been autogenerated by utils/update_llc_test_checks.py UTC_ARGS: --version 5
+; RUN: llc -mtriple=arm-linux-gnueabi -mcpu=arm1022e -mattr=armv5te -o - %s | FileCheck %s
```

Comment:
Pretty sure this would need to be +armv5te, does this print a parse warning?

Reply:
It's quite weird as I didn't get any warning, this definitely needs to be fixed. Dropped it as turned to be not needed (gets exercized anyways).